src/test/java/com/recurly/v3/PagerTest.java

```diff
@@ -165,4 +166,88 @@ private String getResourceSecondPageJson() {
   private String getEmptyListJson() {
     return "{\"object\":\"list\",\"has_more\":false,\"next\":null,\"data\":[]}";
   }
+
+  private String getResourceFirstItemJson() {
+    return "{" +
+      "\"object\": \"list\"," +
+      "\"has_more\": false," +
+      "\"next\": null," +
+      "\"data\": [" +
+      "  {" +
+      "    \"my_string\":\"Resource First Item\"" +
+      "  }" +
+      "]}";
+  }
+
+  private final Response buildResponse(Integer code, String message, String response) {
+    Headers headers = new Headers.Builder().build();
+    return buildResponse(code, message, response, headers);
+  }
+
+  private final Response buildResponse(Integer code, String message, String response, Headers headers) {
+    final Request mRequest = new Request.Builder().url("https://v3.recurly.com").build();
+
+    final Response mResponse =
+        new Response.Builder()
+            .request(mRequest)
+            .protocol(okhttp3.Protocol.HTTP_1_1)
+            .code(code) // status code
+            .message(message)
+            .body(ResponseBody.create(MediaType.get("application/json; charset=utf-8"), response))
+            .headers(headers)
+            .build();
+    return mResponse;
+  }
+
+  private static OkHttpClient getMockOkHttpClient(Answer answer) {
+    final OkHttpClient mockOkHttpClient = mock(OkHttpClient.class);
+    doAnswer(answer).when(mockOkHttpClient).newCall(any());
+    return mockOkHttpClient;
+  }
+
+  @Test
+  public void testCount() {
+    final Call mCall = mock(Call.class);
+    Headers headers = new Headers.Builder().set("Recurly-Total-Records", "1337").build();
+    Answer answer = (i) -> {
+      Request request = i.getArgument(0);
+      assertEquals("HEAD", request.method());
+      return mCall;
+    };
+    try {
+      when(mCall.execute()).thenReturn(buildResponse(200, "OK", getResourceFirstItemJson(), headers));
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Shouldn't we just let this explode and fail the test? or we need to add a fail call here.